Source/WebKit/UIProcess/WebPageProxy.cpp

```diff
@@ -891,6 +891,9 @@ void WebPageProxy::launchProcess(const RegistrableDomain& registrableDomain, Pro
     if (m_preferences->store().getBoolValueForKey(WebPreferencesKey::ipcTestingAPIEnabledKey()))
         m_process->setIgnoreInvalidMessageForTesting();
 #endif
+    
+    if (m_preferences->allowTestOnlyIPC())
```

Comment:
Get this from the configuration instead of preferences.  Then remove it from the yaml